scene/gui/code_edit.cpp

```diff
@@ -2959,9 +2959,16 @@ void CodeEdit::_filter_code_completion_candidates_impl() {
 		}
 
 		if (in_string != -1) {
+			// The completion string may have a literal behind it, which should be removed before re-quoting.
+			String literal = "";
```

Comment:
Strings are already empty by default.
```suggestion
			String literal;
```